diff --git a/modules/juce_core/memory/juce_Memory.h b/modules/juce_core/memory/juce_Memory.h
index 314cfa8..3923657 100644
--- a/modules/juce_core/memory/juce_Memory.h
+++ b/modules/juce_core/memory/juce_Memory.h
@@ -27,12 +27,6 @@
 #define __JUCE_MEMORY_JUCEHEADER__
 
 //==============================================================================
-#if JUCE_MINGW
- /** This allocator is not defined in mingw gcc. */
- #define alloca              __builtin_alloca
-#endif
-
-//==============================================================================
 /** Fills a block of memory with zeros. */
 inline void zeromem (void* memory, size_t numBytes) noexcept        { memset (memory, 0, numBytes); }
 
diff --git a/modules/juce_core/native/juce_BasicNativeHeaders.h b/modules/juce_core/native/juce_BasicNativeHeaders.h
index 43451bb..074506f 100644
--- a/modules/juce_core/native/juce_BasicNativeHeaders.h
+++ b/modules/juce_core/native/juce_BasicNativeHeaders.h
@@ -83,7 +83,11 @@
 
  #define STRICT 1
  #define WIN32_LEAN_AND_MEAN 1
- #define _WIN32_WINNT 0x0600
+ #if JUCE_MINGW
+  #define _WIN32_WINNT 0x0501
+ #else
+  #define _WIN32_WINNT 0x0600
+ #endif
  #define _UNICODE 1
  #define UNICODE 1
  #ifndef _WIN32_IE
diff --git a/modules/juce_core/native/juce_win32_ComSmartPtr.h b/modules/juce_core/native/juce_win32_ComSmartPtr.h
index d4373b9..0bee056 100644
--- a/modules/juce_core/native/juce_win32_ComSmartPtr.h
+++ b/modules/juce_core/native/juce_win32_ComSmartPtr.h
@@ -130,12 +130,7 @@ public:
 
     JUCE_COMRESULT QueryInterface (REFIID refId, void** result)
     {
-       #if ! JUCE_MINGW
         if (refId == __uuidof (ComClass))   { AddRef(); *result = dynamic_cast <ComClass*> (this); return S_OK; }
-       #else
-        jassertfalse; // need to find a mingw equivalent of __uuidof to make this possible
-       #endif
-
         if (refId == IID_IUnknown)          { AddRef(); *result = dynamic_cast <IUnknown*> (this); return S_OK; }
 
         *result = 0;
diff --git a/modules/juce_core/system/juce_TargetPlatform.h b/modules/juce_core/system/juce_TargetPlatform.h
index 919621a..d5211c2 100644
--- a/modules/juce_core/system/juce_TargetPlatform.h
+++ b/modules/juce_core/system/juce_TargetPlatform.h
@@ -81,6 +81,11 @@
 
   #ifdef __MINGW32__
     #define JUCE_MINGW 1
+    #ifdef __MINGW64__
+      #define JUCE_64BIT 1
+    #else
+      #define JUCE_32BIT 1
+    #endif
   #endif
 
   /** If defined, this indicates that the processor is little-endian. */
