diff --git a/modules/juce_gui_basics/juce_gui_basics.cpp b/modules/juce_gui_basics/juce_gui_basics.cpp
index 2e03353..31ef851 100644
--- a/modules/juce_gui_basics/juce_gui_basics.cpp
+++ b/modules/juce_gui_basics/juce_gui_basics.cpp
@@ -85,7 +85,7 @@
  #endif
 
  #if JUCE_MINGW
-  #include <Imm.h>
+  #include <imm.h>
  #endif
 
 //==============================================================================
diff --git a/modules/juce_gui_basics/native/juce_win32_Windowing.cpp b/modules/juce_gui_basics/native/juce_win32_Windowing.cpp
index 3cd2e77..2fd8e83 100644
--- a/modules/juce_gui_basics/native/juce_win32_Windowing.cpp
+++ b/modules/juce_gui_basics/native/juce_win32_Windowing.cpp
@@ -54,7 +54,11 @@ void* getUser32Function (const char* functionName) // (NB: this function also us
     HMODULE user32Mod = GetModuleHandleA ("user32.dll");
     jassert (user32Mod != 0);
 
+#if JUCE_MINGW
+    return (void*)GetProcAddress (user32Mod, functionName);
+#else
     return static_cast <void*> (GetProcAddress (user32Mod, functionName));
+#endif
 }
 
 //==============================================================================
